dom5.ts

```diff
@@ -12,13 +12,10 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-/// <reference path="./custom_typings/main.d.ts" />
 import * as cloneObject from 'clone';
-import * as parse5 from 'parse5';
-import {ASTNode as Node} from 'parse5';
-export {ASTNode as Node} from 'parse5';
```

Comment:
Stepped back this change for another day/PR. But it does sound like this is the direction we'd like to go, now that we're asking users to use parse5 more directly? At the very least, I'd rather export as `ASTNode` and not our special `Node` shortcut.